data/.wrglsync.yaml

```diff
@@ -0,0 +1,618 @@
+      - event_uid
+  event-shreveport-pd:
+    fileName: fuse/event_shreveport_pd.csv
+    primaryKey:
+      - event_uid
+  event-lafayette-so:
+    fileName: fuse/event_lafayette_so.csv
+    primaryKey:
+      - event_uid
+  event-ponchatoula-pd:
+    fileName: fuse/event_ponchatoula_pd.csv
+    primaryKey:
+      - event_uid
+  event-lafayette-pd:
+    fileName: fuse/event_lafayette_pd.csv
+    primaryKey:
+      - event_uid
+  event-hammond-pd:
+    fileName: fuse/event_hammond_pd.csv
+    primaryKey:
+      - event_uid
+  event-sterlington-pd:
+    fileName: fuse/event_sterlington_pd.csv
+    primaryKey:
+      - event_uid
+  event-youngsville-pd:
+    fileName: fuse/event_youngsville_pd.csv
+    primaryKey:
+      - event_uid
+  event-west-monroe-pd:
+    fileName: fuse/event_west_monroe_pd.csv
+    primaryKey:
+      - event_uid
+  event-carencro-pd:
+    fileName: fuse/event_carencro_pd.csv
+    primaryKey:
+      - event_uid
+  event-central-csd:
+    fileName: fuse/event_central_csd.csv
+    primaryKey:
+      - event_uid
+  event-bossier-city-pd:
+    fileName: fuse/event_bossier_city_pd.csv
+    primaryKey:
+      - event_uid
+  event-baker-pd:
+    fileName: fuse/event_baker_pd.csv
+    primaryKey:
+      - event_uid
+  event-houma-pd:
+    fileName: fuse/event_houma_pd.csv
+    primaryKey:
+      - event_uid
+  event-gonzales-pd:
+    fileName: fuse/event_gonzales_pd.csv
+    primaryKey:
+      - event_uid
+  event-denham-springs-pd:
+    fileName: fuse/event_denham_springs_pd.csv
+    primaryKey:
+      - event_uid
+  event-abbeville-pd:
+    fileName: fuse/event_abbeville_pd.csv
+    primaryKey:
+      - event_uid
+  event-washington-so:
+    fileName: fuse/event_washington_so.csv
+    primaryKey:
+      - event_uid
+  event-cameron-so:
+    fileName: fuse/event_cameron_so.csv
+    primaryKey:
+      - event_uid
+  event-maurice-pd:
+    fileName: fuse/event_maurice_pd.csv
+    primaryKey:
+      - event_uid
+  event-terrebonne-so:
+    fileName: fuse/event_terrebonne_so.csv
+    primaryKey:
+      - event_uid
+  event-post:
+    fileName: fuse/event_post.csv
+    primaryKey:
+      - event_uid
+  personnel:
+    fileName: fuse/personnel.csv
+    primaryKey:
+      - uid
+  event:
+    fileName: fuse/event.csv
+    primaryKey:
+      - event_uid
+  allegation:
+    fileName: fuse/allegation.csv
+    primaryKey:
+      - allegation_uid
+  use-of-force:
+    fileName: fuse/use_of_force.csv
+    primaryKey:
+      - uof_uid
+  stop-and-search:
+    fileName: fuse/stop_and_search.csv
+    primaryKey:
+      - stop_and_search_uid
+  baton-rouge-da-brady-list:
+    fileName: match/cprr_baton_rouge_da_2021.csv
+  new-orleans-da-brady-list:
+    fileName: match/cprr_new_orleans_da_2021.csv
+  person:
+    fileName: match/person.csv
+    primaryKey:
+      - person_id
+commit:
+  messageTemplate: |-
+    {{.GitCommit.Link}}
+    {{.GitCommit.Message}}
+  gitRemote:
+    host: github.com
+    dir: ppact/processing
+  gitRoot: ..
```

Comment:
This file shouldn't be here. Have you merged the main branch into this branch yet?

Reply:
I have merged main into this branch. No idea why its still here. This file is no longer present in the other branches that I'm working from where I've merged main, i.e., the `allegations/houma_pd_2017_2018` branch that you approved yesterday.

Reply:
I just merged the `allegations/houma_pd_2017_2018` branch with main, which itself correctly merged with main prior to pushing. After pushing `allegations/houma_pd_2017_2018`  to main, I again, as you'll see below, merged this branch with main, however, data/.wrgylsync.yaml is still present.

Reply:
I figured it out because I just made the same mistake in the `enhancement/allegations_acadia_so branch`. 

Instead of proceeding with the `data/.wrgylsync.yaml` as deleted by you, I re-added the file and then merged. As you'll see in that branch, I deleted the  `data/.wrgylsync.yaml` file and added all relevant data to `wrgl/config.yaml `. 

In this branch, I've also just removed`data/.wrgylsync.yaml` and added post to `wrgl/config.yaml`

Reply:
It appears alright now :)

Reply:
Awesome.